packages/react-native-testing-library/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@testing-library/react-native",
+  "version": "7.0.1",
+  "description": "Simple and complete React Native testing utilities that encourage good testing practices.",
+  "main": "build/index.js",
+  "typings": "./typings/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://www.github.com/callstack/react-native-testing-library.git"
+  },
+  "homepage": "https://callstack.github.io/react-native-testing-library",
+  "author": "Michał Pierzchała <thymikee@gmail.com>",
+  "license": "MIT",
+  "private": false,
+  "keywords": [
+    "react-native",
+    "react",
+    "test",
+    "integration"
+  ],
+  "files": [
+    "build/",
+    "typings/index.d.ts",
+    "pure.js",
+    "dont-cleanup-after-each.js"
+  ],
+  "devDependencies": {
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.9.0",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/preset-env": "^7.9.6",
+    "@babel/preset-flow": "^7.9.0",
+    "@babel/preset-react": "^7.9.4",
+    "@callstack/eslint-config": "*",
+    "@release-it/conventional-changelog": "^1.1.0",
+    "@testing-library/jest-native": "~3.3.0",
+    "@types/react": "*",
+    "@types/react-native": "*",
+    "@types/react-test-renderer": "*",
+    "babel-jest": "*",
+    "conventional-changelog-cli": "^2.0.11",
+    "dedent": "^0.7.0",
+    "eslint": "*",
+    "flow-bin": "^0.122.0",
+    "flow-copy-source": "^2.0.9",
+    "jest": "*",
+    "react": "*",
+    "react-native": "*",
+    "react-test-renderer": "*",
```

Comment:
can we get rid of `*` in our dev deps and bring back the versions we had in original package.json? I don't mind having `*` in examples though